File: Data Structures & Algorithms/duplicate-integer/submission-1.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        Set<Integer> letter = new HashSet<>();
        int res = 0;
        for(int i = 0; i<nums.length; i++){
            if(letter.contains(nums[i])){
                return true;
            }
            letter.add(nums[i]);
        }
        return false;
    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-1.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        nums = set(nums)
        res = 0
        for num in nums:
            count = 1
            while num + 1 in nums:
                count +=1
                num +=1
            res = max(res,count)
        return res